Automattic/woocommerce-payments
# Filtering APMs by billing country

Fixes #7752

#### Changes proposed in this Pull Request

Updated the list of countries for APMs according to the list provided by Stripe Docs.

#### Testing instructions

Check the list of countries available for each payment method in [Stripe Docs](https://stripe.com/docs/payments/bancontact) ([SEPA](https://stripe.com/en-br/resources/more/sepa-country-list#list-of-sepa-countries) is on separate page) and ensure they are correctly mapped and working correctly according to the selected billing country.

-------------------


**Post merge**
